src/main/java/com/unmoon/ToolRequiredPlugin.java

```diff
@@ -4,20 +4,16 @@
 import com.google.inject.Provides;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.Client;
```

Comment:
Please don't change the import style. You can recreate my style with the following settings, and running Code -> Optimize Imports (Ctrl+Alt+O) in IntelliJ IDEA.
<img width="1012" height="755" alt="image" src="https://github.com/user-attachments/assets/e3c18908-eb0f-457a-aaf7-be3a8ac52377" />